src/components/UserAccess/UserAccessForm/form-utils.ts

```diff
@@ -30,29 +20,60 @@ export const userAccessFormSchema = yup.object({
     .required('Required.'),
 });
 
-export const createSBRs = async (
+export const getRBs = (namespace: string, dryRun?: boolean): Promise<RoleBinding[]> => {
+  return K8sListResourceItems({
+    model: RoleBindingModel,
+    queryOptions: {
+      ns: namespace,
+      ...(dryRun && { queryParams: { dryRun: 'All' } }),
+    },
+  })
+    .then((res) => {
+      const roleBindings: RoleBinding[] = res as RoleBinding[];
+      return roleBindings;
+    })
+    .catch((error) => {
+      // Handle error if needed
+      throw error;
+    });
+};
+
+/**
+ * Create role-bindings to shared konflux cluster roles
+ */
+export const createRBs = async (
   values: UserAccessFormValues,
   namespace: string,
   dryRun?: boolean,
-): Promise<SpaceBindingRequest[]> => {
+): Promise<RoleBinding[]> => {
   const { usernames, role } = values;
-  const objs: SpaceBindingRequest[] = usernames.map((username) => ({
-    apiVersion: `${SpaceBindingRequestGroupVersionKind.group}/${SpaceBindingRequestGroupVersionKind.version}`,
-    kind: SpaceBindingRequestGroupVersionKind.kind,
+  const roleRefName = `konflux-${role}-user-actions`;
```

Comment:
Dont hard code the role ref name, selected role should already contain this value.